src/OpenTelemetry.Instrumentation.AspNetCore/MeterProviderBuilderExtensions.cs

```diff
@@ -61,6 +79,16 @@ public static class MeterProviderBuilderExtensions
     {
         Guard.ThrowIfNull(builder);
 
+#if NET8_0_OR_GREATER
+        return builder
```

Comment:
It'd be good to log a warning here saying the configurations for `AspNetCoreInstrumentationOptions` would not be applied on `NET8_0_OR_GREATER`.

Reply:
Added: Note, this will be removed in follow up PR which will remove the options entirely from metrics.